evaluation/evaluation_metrics/evaluate_generation/generation_metrics/README.md

```diff
@@ -0,0 +1,151 @@
+
+# Generation Evaluation Metrics
+
+This project is designed to evaluate the generation of responses using language models (LLMs) in a Retrieval-Augmented Generation (RAG) setup. This README provides the necessary steps to set up and run the project, including executing several scripts that perform specific tasks.
+
+## Prerequisites
+
+### 1. Create a Virtual Environment
+Before starting, you need to create two virtual environments with Python 3.9 and install the required packages:
+
+```bash
+python3.9 -m venv venv-dev
+source venv-dev/bin/activate
+pip install -r requirements-dev.txt
+
+python3.9 -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
```

Comment:
Why is it needed to have 2 environments instead of 1? Are there conflicting library versions?

Reply:
I considered unnecesary to install boto3, clearml, python-dotenv, pandas, tqdm and tiktoken in the main environment, what do you think?